Make.inc

```diff
@@ -125,7 +125,33 @@ else
 NO_GIT := 1
 endif
 
+# Julia's Semantic Versioning system labels the three decimal places in a version number as
+# the major, minor and patch versions.  Typically the major version would be incremented
+# whenever a backwards-incompatible change is made, the minor version would be incremented
+# whenever major backwards-compatible changes are made, and the patch version would be
+# incremented whenever smaller changes are made.  However, before v1.0.0, the major
+# version number is always zero and the meanings shift down a place; the minor version
+# number becomes the major version number, the patch version number becomes the minor
+# version number, and there is no patch version number to speak of.  In this case, the
+# version v0.4.1 has major backwards-compatible changes as compared to v0.4.0, and the
```

Comment:
Well, what I'm going for there is that with the `Major.Minor.Patch` standard, `Major` is incremented for large backwards-incompatible changes, `Minor` is incremented for large backwards-compatible changes, and `Patch` is used for small backwards-compatible changes.  Because before `v1.0.0` all the meanings shift down by one decimal place, that means that this line should hopefully be accurate in explaining that the difference between `v0.4.0` and `v0.4.1` is a `Minor` bump, not a `Patch` bump.